File: codewars/Anagram difference/Anagram difference.py
def anagram_difference(w1, w2):
    count_w1 = [0] * 26
    count_w2 = [0] * 26
    for char in w1:
        count_w1[ord(char) - ord('a')] += 1
    for char in w2:
        count_w2[ord(char) - ord('a')] += 1
    deletions = sum(abs(count_w1[i] - count_w2[i]) for i in range(26))
    return deletions

File: codewars/Anagram difference/AnagramDifference.js
function anagramDifference(w1, w2) {
    let count = [];
    for (let i = 0; i < w1.length; i++) {
        count[w1[i]] = (count[w1[i]] || 0) + 1;
    }
    for (let i = 0; i < w2.length; i++) {
        count[w2[i]] = (count[w2[i]] || 0) - 1;
    }
    let answer = 0;
    for (let x in count) {
        answer += Math.abs(count[x])
    }
    return answer
}
